### PR TITLE
fix: character count is not shown proper for textarea.

### PR DESCRIPTION
**Pull request by shakil-mansuri-dream**

### CODE REVIEW

File: dw-input.js [hunk 1]
```diff
@@ -640,7 +640,7 @@ export class DwInput extends DwFormElement(LitElement) {
         .minHeight="${this.minHeight}"
         minlength="${this.minLength}"
```
Comment (ChiragMoradiya) [inline dw-input.js:641]: Isn't there a similar issue for `minLength`?

Comment (shakil-mansuri-dream) [reply dw-input.js:641]: @ChiragMoradiya हा हे, में इसको भी fix कर देता हु.

Comment (ChiragMoradiya) [reply dw-input.js:641]: ok. Thanks

Comment (shakil-mansuri-dream) [reply dw-input.js]: @ChiragMoradiya fixed

**Pull request merged**